server/aurora_mcp/tools_gated.py

```
@@ -0,0 +1,333 @@
+            "query": query,
+            "earliestTime": f"-{int(time_range_minutes)}m",
+            "latestTime": "now",
+            "maxCount": limit,
+        }
+        return truncate_payload(
+            await api_call("POST", "/splunk/search", body=body),
+            tool_name="query_logs",
+        )
+    # Unreachable today — fires only if TIER2_TOOLS["query_logs"].enabling_skills
+    # is extended without a matching branch above. Raise to make the gap loud.
+    raise AssertionError(f"query_logs: no dispatch branch for source {chosen!r}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)